### ISSUE TITLE
Update Swagger to the latest version

### ISSUE THREAD

**Issue by mcicolella** [freedomotic/freedomotic#452]
Our plugin [RestAPI ](https://github.com/freedomotic/freedomotic/blob/2f734aeb8b58b433c8ccb3b4eed8b2fd6ed2adb1/plugins/devices/restapi-v3/pom.xml)uses Swagger to document the APIs.
We would update Swagger to the latest available version.

**Comment by basalt79** [comment]
i will have a look on the src and give it a try

**Comment by lubarinobr** [comment]
I'd like work in this issue.

**Comment by mcicolella** [comment]
@alejandrocq check this PR #457 sent by @marcopollivier. He updated swagger-maven-plugin to 3.1.7 without making any changes to the plugin configuration.

**Comment by alejandrocq** [comment]
> @alejandrocq check this PR #457 sent by @marcopollivier. He updated swagger-maven-plugin to 3.1.7 without making any changes to the plugin configuration.

Yes, I have checked it, but compilation fails because some configuration parameters of the plugin have changed. For example, title and version of the API must be included inside <info> tag. You can see the latest configuration parameters here: https://github.com/kongchen/swagger-maven-plugin

As I said, I can do a PR later so you can see my changes :)

**Comment by marcopollivier** [comment]
Hello. I had misunderstood the issue. I'm sorry. So... I'm glad that the speedy guy has resolved before I could.

I will delete the PR

**Comment by marcopollivier** [comment]
Hello. I had misunderstood the issue. I'm sorry. So... I'm glad that has resolved before I could.

I will delete the PR

**Comment by GDSGUI** [comment]
Boa Tarde eu estava assistindo as suas aulas de orientação a objetos Java e vim no git Hub para ter acesso aos códigos da aula más não estou achando, eles não estão mais desooniveis?
